tests/test_arraycreation.py

```diff
@@ -552,7 +540,9 @@ def test_full(shape, fill_value, dtype, order):
     "fill_value", [1.5, 2, 1.5 + 0.0j], ids=["1.5", "2", "1.5+0.j"]
 )
 @pytest.mark.parametrize("dtype", get_all_dtypes(no_float16=False))
-@pytest.mark.parametrize("order", [None, "C", "F"], ids=["None", "C", "F"])
+@pytest.mark.parametrize(
+    "order", [None, "C", "F", "A", "K"], ids=["None", "C", "F", "A", "K"]
+)
```

Comment:
It will result into the same output (the same name of test):
```suggestion
@pytest.mark.parametrize("order", [None, "C", "F", "A", "K"])
```